mvp.feature

```diff
@@ -144,27 +144,27 @@ Feature: User with no Author identity in the Authority Register for Personas see
     And they see that the information for Publisher is correct
```

Comment:
Hva er `eISSN`? Vi har onlineISSN og printISSN.
Finnes ikke publishers for Journals. Hva er publisher da?

Reply:
Tror eISSSN er onlineISSN